target/linux/generic/backport-5.15/781-v6.1-bus-mhi-host-always-print-detected-modem-name.patch

```diff
@@ -21,11 +21,11 @@ Link: https://lore.kernel.org/r/20220831100349.1488762-1-koen.vandeputte@citymes
 [mani: added missing review tags]
 Signed-off-by: Manivannan Sadhasivam <manivannan.sadhasivam@linaro.org>
 ---
- drivers/bus/mhi/pci_generic.c | 2 +-
+ drivers/bus/mhi/host/pci_generic.c | 2 +-
  1 file changed, 1 insertion(+), 1 deletion(-)
 
---- a/drivers/bus/mhi/pci_generic.c
-+++ b/drivers/bus/mhi/pci_generic.c
+--- a/drivers/bus/mhi/host/pci_generic.c
++++ b/drivers/bus/mhi/host/pci_generic.c
```

Comment:
Please adapt the file `package/kernel/linux/modules/other.mk` too.

Reply:
I did this in the 1st commit of this PR.  Would you like them squashed into a single commit?
https://github.com/openwrt/openwrt/pull/12641/commits/58bb9d13c89db83b9d449fb428db511cbcd8eef8